### PR TITLE
Refactor html output

### PR DESCRIPTION
**Pull request by jonathanyeong**
This pull request takes out the static html from trello_newsletter.rb and puts it into html_stitcher.rb. It also adds some tests to make sure index.html is structurally sound (has head and body tags).

### CODE REVIEW

File: lib/html_stitcher.rb [hunk 1]
```diff
@@ -0,0 +1,679 @@
+class HtmlStitcher
+  def head
+    head_string = <<-DOC.gsub(/^ {4}/, '')
```
Comment (sarony) [inline lib/html_stitcher.rb]: Why do we need to set this local variable?

File: lib/html_stitcher.rb [hunk 2]
```diff
@@ -0,0 +1,679 @@
...
+              }
+            }
+          }
+        </style>
+      </head>
+    DOC
+  end
+
+  def body_start(meta)
+    body_string = <<-DOC.gsub(/^ {4}/, '')
```
Comment (sarony) [inline lib/html_stitcher.rb]: Same comment as above. Why the local variable?

File: lib/html_stitcher.rb [hunk 3]
```diff
@@ -0,0 +1,679 @@
...
+                                        <tr>
+                                              <td valign="top" class="bodyContent">
+                                                  <!-- // Begin Module: Standard Content \\ -->
+                                                  <table border="0" cellpadding="20" cellspacing="0" width="100%">
+                                                    <tr>
+    DOC
+  end
+
+  def body_end
+    body_string = <<-DOC.gsub(/^ {4}/, '')
```
Comment (sarony) [inline lib/html_stitcher.rb]: Same comment as above on the local variable

File: spec/views/index.html_spec.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+require 'nokogiri'
+
+describe "html output" do
```
Comment (sarony) [inline spec/views/index.html_spec.rb:4]: good use of Nokogiri!

**Comment by sarony** [comment]
Let me know about those local variables, I don't think we need them. Otherwise, looks good!

**Comment by jonathanyeong** [comment]
I'm using the local variables to store the heredoc string and return it when the html_stitcher is called. I wasn't sure if it would still return the heredoc string if you take the local variables out. I'll see if it works.

**Comment by jonathanyeong** [comment]
@sarony updated the local variables. Finished adding the rest of the newsletter to the HTML factory. Let me know if I can change anything when we meet next.

**Pull request merged**